service/controller/resource/pvc/resource.go

```diff
@@ -13,6 +13,7 @@ const (
 	// StorageClass is the storage class annotation persistent volume claims are
 	// configured with.
 	StorageClass = "g8s-storage"
+	LocalStorageClass = "local-storage"
```

Comment:
We should create beforehand this storage class. I have looked at that but it is not the k8scloudconfig that create `StorageClass`.